src/moin/templates/base.html

```diff
@@ -53,10 +53,9 @@
 
                 {%- block theme_stylesheets %}
                     <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/normalize.css') }}" />
+                    <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/variables.css') }}" />
                     {% if theme_supp.variables_css() %}
                         <link media="all" rel="stylesheet" href="{{ theme_static('css/variables.css') }}" />
-                    {% else %}
-                        <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/variables.css') }}" />
```

Comment:
guess the idea of the old code was that a theme's variables.css may or may not @import the default variables.css.

Reply:
My understanding is browsers load CSS files in parallel, but @import files cannot be loaded until CSS parsing begins. Small benefit.

Old code loaded only 1 variables.css assuming that if a theme had a variables.css it was going to override all the common.css colors; maybe implement a dark theme. New code allows a theme to have a variables.css if it wants one.